Base/usr/share/man/man2/module_load.md

```diff
@@ -0,0 +1,35 @@
+## Name
+
+module\_load - load a kernel module
+
+## Synopsis
+
+```**c++
+#include <serenity.h>
+
+int module_load(const char* name, size_t name_length);
+```
+
+## Description
+
+`module_load()` will load a kernel module from an ELF object file given its
+path in the filesystem.
```

Comment:
So is this a name or a path?

Reply:
It's a path - the typo was in the prototype.